DataFormats/Detectors/CTP/include/DataFormatsCTP/Digits.h

```diff
@@ -20,30 +24,33 @@ namespace o2
 {
 namespace ctp
 {
-static constexpr uint32_t CTP_NINPUTS = 46;
-static constexpr uint32_t CTP_NCLASSES = 64;
-static constexpr uint32_t CTP_MAXL0PERDET = 5;
-// Positions of CTP Detector inputs in CTPInputMask
+/// CTP related constants
+static constexpr std::uint32_t NumOfHBInTF = 256;
+static constexpr std::uint32_t CTP_NINPUTS = 46;    /// Max number of CTP inputs for all levels
+static constexpr std::uint32_t CTP_NCLASSES = 64;   /// Number of classes in hardware
+static constexpr std::uint32_t CTP_MAXL0PERDET = 5; /// Max number of LM/L0inputs per detector
+/// Positions of CTP Detector inputs in CTPInputMask
 static constexpr std::pair<uint32_t, std::bitset<CTP_MAXL0PERDET>> CTP_INPUTMASK_FV0(0, 0x1f);
 static constexpr std::pair<uint32_t, std::bitset<CTP_MAXL0PERDET>> CTP_INPUTMASK_FT0(5, 0x1f);
+///
 struct CTPDigit {
-  o2::InteractionRecord mIntRecord;
-  std::bitset<CTP_NINPUTS> mCTPInputMask;
-  std::bitset<CTP_NCLASSES> mCTPClassMask;
+  o2::InteractionRecord intRecord;
+  std::bitset<CTP_NINPUTS> CTPInputMask;
+  std::bitset<CTP_NCLASSES> CTPClassMask;
   CTPDigit() = default;
   void printStream(std::ostream& stream) const;
   ClassDefNV(CTPDigit, 1);
 };
 struct CTPInputDigit {
-  o2::InteractionRecord mIntRecord;
-  std::bitset<CTP_MAXL0PERDET> mInputsMask;
-  std::int32_t mDetector;
+  o2::InteractionRecord intRecord;
+  std::bitset<CTP_MAXL0PERDET> inputsMask;
+  std::int32_t detector;
   CTPInputDigit() = default;
-  CTPInputDigit(o2::InteractionRecord IntRecord, std::bitset<CTP_MAXL0PERDET> InputsMask, uint32_t DetID)
+  CTPInputDigit(o2::InteractionRecord intRecord, std::bitset<CTP_MAXL0PERDET> inputsMask, std::uint32_t detID)
```

Comment:
you actually even don't need this c-tor for such a trivial classes, e.g. you can create an instance simply via initialization list
````
CTPInputDigit finpdigit{inp.mIntRecord, inp.mInputs, o2::detectors::DetID::FT0};
````
If you wan to keep it, make argument names different from data members

Reply:
removed

Reply:
@shahor02 I did fixes. There is one open question on logic you suggested.